src/screens/SignupScreen.tsx

```diff
@@ -292,11 +295,19 @@ const SignupScreen = () => {
                             >
                                 Sign up
                             </Button>
+                            
                         </VStack>
+                        <Image
+                    
+                                source={require('../../assets/IconTransparent.png')}
+                                style={[styles.container, styles.imageContainer]}
```

Comment:
Change this line to `size="lg"` or `size="md"`, whatever looks better. See here for more info: https://docs.nativebase.io/image#h3-sizes . By doing this, the image should resize based on breakpoints defined by Native Base